### PR TITLE
Add preference toggling touch events.

### PR DESCRIPTION
**Pull request by frewsxcv**
Fixes https://github.com/servo/servo/issues/11477.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/14534)

### CODE REVIEW

File: resources/prefs.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
   "dom.serviceworker.timeout_seconds": 60,
   "dom.testable_crash.enabled": false,
   "dom.testbinding.enabled": false,
+  "dom.w3c_touch_events.enabled": true,
```
Comment (frewsxcv) [inline resources/prefs.json]: Pref key name is borrowed from Firefox.

**Comment by frewsxcv** [comment]
Rebased. Conflicts have been addressed.

File: components/script/dom/document.rs [hunk 2]
```diff
@@ -2571,7 +2571,7 @@ impl DocumentMethods for Document {
                 let USVString(url) = self.URL();
                 Ok(Root::upcast(StorageEvent::new_uninitialized(&self.window, DOMString::from(url))))
             },
-            "touchevent" =>
+            "touchevent" if PREFS.is_touch_events_enabled() =>
```
Comment (shinglyu) [inline components/script/dom/document.rs]: Is it enough to handle it here? From [`compositor.rs`](https://github.com/servo/servo/blob/master/components/compositing/compositor.rs#L813) there seem to be another path that will eventually call `document.handle_touch_event()` without going though here?

**Comment by jdm** [comment]
Closing due to lack of activity.

**Pull request closed without merging**